vars/sample-acs-team-access-config.yaml

```diff
@@ -11,13 +11,22 @@ access_scope:
     rules:
       includedClusters: []
       includedNamespaces: []
-      clusterLabelSelectors: []
+      clusterLabelSelectors: # registry will check duplication
+        - requirements:
```

Comment:
remove this file since it's a duplicate of `vars/acs-team-access-config.yaml`

Reply:
Removed